### PR TITLE
Add openshift/cloud-provider-vsphere repository

### PR DESCRIPTION
**Pull request by lobziik**
- Add Vsphere CCM mages build tasks in CI
  - Enable unit-tests and verify jobs for cloud-provider-vsphere
  - Add OWNERS for cloud-provider-vsphere

### CODE REVIEW

File: core-services/prow/02_config/openshift/cloud-provider-vsphere/_pluginconfig.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+approve:
+- commandHelpLink: ""
+  repos:
+  - openshift/cloud-provider-vsphere
+  require_self_approval: false
```
Comment (JoelSpeed) [inline core-services/prow/02_config/openshift/cloud-provider-vsphere/_pluginconfig.yaml:5]: Do we want want to change this?
```suggestion
  require_self_approval: true
```

Comment (lobziik) [reply core-services/prow/02_config/openshift/cloud-provider-vsphere/_pluginconfig.yaml:5]: Hm... Don't know. 

If we want, lets change it in other our repos as well. Might be done in separate PR, wdyt?

**Comment by smarterclayton** [comment]
/approve

For image mirror name

**Pull request merged**